_episodes/02-shell-basics.md

```diff
@@ -1,8 +1,20 @@
 ---
-title: "Shell basics"
+title: "Introduction to the Shell"
 questions:
 - "How to look at files and folders using the shell?"
 - "How are files and folders organized on this computer?"
 - "How to create folders and files?"
 - "How to specify the location of a file or folder?"
+- "What is a shell script and how do I write one?"
```

Comment:
Going from learning basic commands to navigate and manipulate the file system (`ls`, `pwd`, `cd`, `rm`, `mv`, `cp`) to writing shell scripts is a **big** leap for learners. Also, they may not see value in a script that just  uses these commands.

I think that some core ideas need to be covered first:

1. The shell is more than a way to look at files or folders. It's also a way to run programs.
1. Programs may expect data as input and may produce data as output. The shell lets us control which programs to run on which inputs and what to do with the output.